docs/user/production-considerations/task-manager-health-monitoring.asciidoc

```diff
@@ -90,6 +90,15 @@ a| Runtime
 
 | This section tracks excution performance of Task Manager, tracking task _drift_, worker _load_, and execution stats broken down by type, including duration and execution results.
 
+a| Capacity Estimation
+
+| This section provides a rough estimate about the sufficiency of its capacity. As the name suggests, these are estimates based on historical data and should not be used as predictions. We recommend using these estimations when following the Task Manager <<task-manager-scaling-guidance>>.
++
+[NOTE]
```

Comment:
This is showing up kind of funky in the docs.

Reply:
Oops looks like you can't put a Note inside of a definitions list 😬